OrbitQt/GgpClient.h

```diff
@@ -0,0 +1,35 @@
+#ifndef ORBITQT_GGP_CLIENT_H_
+#define ORBITQT_GGP_CLIENT_H_
+
+#include <QVector>
+#include <functional>
+#include <outcome.hpp>
+#include <string>
+
+#include "GgpInstance.h"
+
+class GgpClient {
+ public:
+  // this policy means when a result is wrongly accessed, std::terminate() will
+  // be called. (.value() is accessed even though it is an error, or vice versa)
```

Comment:
https://ned14.github.io/outcome/reference/aliases/default_policy/

The default policy is "do not compile" when the error type does not provide an `std::error_code` or exception pointer. If it provides any of those an exception will be thrown. 

I chose terminate, because this should never occur and is similar to the `FAIL_IF` check.